Assets/Scripts/Combat/Combantants/Combatant.cs

```diff
@@ -32,39 +47,94 @@ public void Start()
+        HealthText = HealthBar.GetComponentInChildren<Text>();
+        RedBar.MaxHealth = MaxHealthPoints;
         HealthBar.SetActive(false);
-
+        
         CurrentHealthPoints = MaxHealthPoints;
     }
 
     public void Update()
     {
         if(IsInCombat)
+        {
             UpdateHealthBar();
+            UpdateShieldBar();
+        }
     }
 
     private void UpdateHealthBar()
     {
-        string healthText;
+        RedBar.NewHealth = CurrentHealthPoints;
+        HealthText.text = $"{CurrentHealthPoints} / {MaxHealthPoints}";
+        
+        UpdateHealthBarPosition();
+    }
 
-        if (CurrentShieldPoints > 0)
-            healthText = IsAlive ?
-                        $"{CurrentHealthPoints} / {MaxHealthPoints} Shield: {CurrentShieldPoints} / {MaxShieldPoints}" :
-                        "I've fallen and can't get up";
+    private void UpdateShieldBar()
+    {
+        if (IsShieldActive && CurrentShieldPoints > 0)
+        {
+            ShieldBar.SetActive(true);
+            BlueBar.NewHealth = CurrentShieldPoints;
+            ShieldText.text = $"{CurrentShieldPoints} / {MaxShieldPoints}";
+        
+            UpdateShieldBarPosition();
+        }
+        else if (CurrentShieldPoints <= 0 && IsShieldActive)
+        {
+            ShieldBar.SetActive(false);
+        }
+        // Only instantiates a shield if ability is called
+        else if (CurrentShieldPoints > 0)
+        {
+            IsShieldActive = true;
+            ShieldBar = Instantiate(ShieldBarPrefab);
+            ShieldBar.transform.SetParent(HealthBarUIPanel.transform);
+            BlueBar = ShieldBar.GetComponentInChildren<HealthBar>();
+            ShieldText = ShieldBar.GetComponentInChildren<Text>();
+            BlueBar.NewHealth = CurrentHealthPoints;
+            BlueBar.MaxHealth = MaxShieldPoints;
+            ShieldText.text = $"{CurrentShieldPoints} / {MaxShieldPoints}";
+        }
 
-        else
-            healthText = IsAlive ?
-                $"{CurrentHealthPoints} / {MaxHealthPoints}" :
-                "I've fallen and can't get up";
+
+    }
+
+    private void UpdateHealthBarPosition()
+    {
+        // Replace code with this when collider is added
+        /*
+        Vector3 yOffset = new Vector3 (0, gameObject.GetComponent<Collider>().bounds.size.y * HealthBarYOffset, 0);
+        Vector3 offsetPos = gameObject.transform.position + yOffset;
+        Vector3 relativeScreenPosition = Camera.main.WorldToScreenPoint(offsetPos);
+        HealthBar.transform.position = relativeScreenPosition;
+        */
 
         Vector3 relativeScreenPosition = Camera.main.WorldToScreenPoint(transform.position);
         relativeScreenPosition.y += HealthBarPosition;
         HealthBar.transform.position = relativeScreenPosition;
-        HealthBar.GetComponentInChildren<Text>().text = healthText;
+    }
+
+    private void UpdateShieldBarPosition()
+    {
+        // Replace code with this when collider is added
```

Comment:
as mentioned, look into CharacterController, since it is an extension of the Collider class, and also has its own capsule collider, _somewhere_